lldb/test/Shell/SymbolFile/DWARF/x86/dead-code-filtering.yaml

```diff
@@ -1,3 +1,5 @@
+# XFAIL: target=x86_64-{{.*}}-windows{{.*}}
```

Comment:
This is particularly concerning, given that this test just operates on static binary data (expressed in yaml form). There should be nothing target-specific here.

Maybe we could start by looking at what's making this test in particular fail?

Reply:
We have it green as well
```
********************
PASS: lldb-shell :: SymbolFile/DWARF/x86/dead-code-filtering.yaml (389 of 543)
Exit Code: 0

Command Output (stdout):
--
# RUN: at line 1
c:\build-lldb-native\bin\yaml2obj.exe C:\lldb\test\Shell\SymbolFile\DWARF\x86\dead-code-filtering.yaml > C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\DWARF\x86\Output\dead-code-filtering.yaml.tmp
# executed command: 'c:\build-lldb-native\bin\yaml2obj.exe' 'C:\lldb\test\Shell\SymbolFile\DWARF\x86\dead-code-filtering.yaml'
# note: command had no output on stdout or stderr
# RUN: at line 2
c:\build-lldb-native\bin\lldb-test.exe symbols C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\DWARF\x86\Output\dead-code-filtering.yaml.tmp | c:\build-lldb-native\bin\filecheck.exe C:\lldb\test\Shell\SymbolFile\DWARF\x86\dead-code-filtering.yaml --check-prefix=TEST
# executed command: 'c:\build-lldb-native\bin\lldb-test.exe' symbols 'C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\DWARF\x86\Output\dead-code-filtering.yaml.tmp'
# note: command had no output on stdout or stderr
# executed command: 'c:\build-lldb-native\bin\filecheck.exe' 'C:\lldb\test\Shell\SymbolFile\DWARF\x86\dead-code-filtering.yaml' --check-prefix=TEST
# note: command had no output on stdout or stderr
# RUN: at line 3
c:\build-lldb-native\bin\lldb.exe --no-lldbinit -S C:/build-lldb-native/tools/lldb\test\Shell\lit-lldb-init-quiet C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\DWARF\x86\Output\dead-code-filtering.yaml.tmp -o "image dump line-table a.c" -o "image lookup -n _start" -o "image lookup -n f" -o exit | c:\build-lldb-native\bin\filecheck.exe C:\lldb\test\Shell\SymbolFile\DWARF\x86\dead-code-filtering.yaml
# executed command: 'c:\build-lldb-native\bin\lldb.exe' --no-lldbinit -S 'C:/build-lldb-native/tools/lldb\test\Shell\lit-lldb-init-quiet' 'C:\build-lldb-native\tools\lldb\test\Shell\SymbolFile\DWARF\x86\Output\dead-code-filtering.yaml.tmp' -o 'image dump line-table a.c' -o 'image lookup -n _start' -o 'image lookup -n f' -o exit
# note: command had no output on stdout or stderr
# executed command: 'c:\build-lldb-native\bin\filecheck.exe' 'C:\lldb\test\Shell\SymbolFile\DWARF\x86\dead-code-filtering.yaml'
# note: command had no output on stdout or stderr

--


```